Update UG and DG (#205)
AY2324S2-CS2103T-W11-2/tp: docs/UserGuide.md

##### Possible Failures:

- Incorrect format of parameters

> - Incorrect order of parameters

Reviewer: I thought order does not matter for add command?

> - Incorrect order of parameters.
> - Didn't match minimum parameters requirements (A contact must have name, phone, email, address).

Reviewer 2: ```suggestion
- Not having the minimum parameters (A contact must have name, phone, email, address).
```

> - Didn't match minimum parameters' requirements (A contact must have name, phone, email, address).

Reviewer 2: ```suggestion
- Didn't match minimum parameters requirements (A contact must have name, phone, email, address).
```
